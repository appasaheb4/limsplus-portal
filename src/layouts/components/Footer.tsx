import React from "react";
import { Container, Row, Col } from "reactstrap";

const Footer = () => (
  <footer className="footer">
    <Container fluid>
      <Row className="text-muted">
        <Col xs="6" className="text-left">
          <ul className="list-inline">
            <li className="list-inline-item">
              <span className="text-muted">Support</span>
            </li>
            <li className="list-inline-item">
              <span className="text-muted">Help Center</span>
            </li>
            <li className="list-inline-item">
              <span className="text-muted">Privacy</span>
            </li>
            <li className="list-inline-item">
              <span className="text-muted">Terms of Service</span>
            </li>
          </ul>
        </Col>
        <Col xs="6" className="text-right">
          <p className="mb-0">
            &copy; {new Date().getFullYear()} -{" "}
            <span className="text-muted">Lims Plus</span>
          </p>
        </Col>
      </Row>
    </Container>
  </footer>
);

export default Footer;
