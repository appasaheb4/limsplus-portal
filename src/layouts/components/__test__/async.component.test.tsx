// eslint-disable-next-line folders/match-regex
import React, {Component} from 'react';
import {render, cleanup} from '@testing-library/react';
import ReactDOM from 'react-dom';
import async from '../async.component';
import Footer from '../footer.component';
import {act} from 'react-dom/test-utils';

describe('asyncComponent component', () => {
  it('should render correct', () => {
    act(() => {
      jest.mock('', () => {
        return {
          State: {
            component: jest.fn(),
          },
          component: jest.fn(),
        };
      });
      const hocAsyncComp = async(() => import('../footer.component'));

      expect(hocAsyncComp).toBeDefined();
      expect(hocAsyncComp).toMatchSnapshot();
    });
  });
});
