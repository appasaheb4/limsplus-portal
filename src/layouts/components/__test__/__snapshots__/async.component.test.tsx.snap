// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`asyncComponent component should render correct 1`] = `[Function]`;
